strings.txt

```diff
@@ -17810,3 +17810,601 @@
     sk = od %d
     es = desde %d
     pl = od %d
+
+  [meter]
+    en = m
+    ru = м
+    fr = m
+    da = m
+    id = m
+    ko = m
+    sv = m
+    tr = m
+    uk = м
+    vi = m
+    hu = m
+    de = m
+    fi = m
+    cs = m
+    it = m
+    nb = m
+    zh-Hans = m
+    th = ม.
+    ja = m
+    ro = m
+    ar = متر
+    sk = m
+    es = m
+    pl = m
+    nl = m
+
+  [kilometer]
```

Comment:
Тут сокращённый текст, там полный, разные вещи.
> On Apr 26, 2016, at 4:35 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61085581>:
> 
> > +    de = m
> > +    fi = m
> > +    cs = m
> > +    it = m
> > +    nb = m
> > +    zh-Hans = m
> > +    th = ม.
> > +    ja = m
> > +    ro = m
> > +    ar = متر
> > +    sk = m
> > +    es = m
> > +    pl = m
> > +    nl = m
> > +
> > +  [kilometer]
> Уже есть [kilometres]
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/454ca48abdc1a7a643a178fac869c249f100121f..9460d6ca1e36b2c17bee62d5d712319e41eb236f#r61085581>